Dockerfile

```diff
@@ -21,4 +21,11 @@ RUN wget -q https://github.com/AsamK/signal-cli/releases/download/v${SIGNAL_CLI_
     ln -sf /opt/signal-cli-${SIGNAL_CLI_VERSION}/bin/signal-cli /usr/local/bin/
 
 WORKDIR /app
-COPY scripts/register-number.sh /app
\ No newline at end of file
+COPY scripts/register-number.sh /app
```

Comment:
Should this script be ran _in_ the docker container prior to running the Python script?

Reply:
Yes, this needs to be run prior to actually starting the container. I'll add some example commands to the README for setting it up!